api/openapi.yaml

```diff
@@ -467,6 +467,36 @@ paths:
           $ref: "#/components/responses/401"
         404:
           $ref: "#/components/responses/404"
+  /users/refreshtoken:
+    put:
+      tags:
+        - "Users"
+      description: Update refresh token.
+      summary: "Silent refresh of tokens"
+      security:
+        - cookieAuth: []
+      requestBody:
```

Comment:
actually, this code needs to be removed, as I mentioned to you earlier Open API does not support cookies by default. /refreshToken endpoint will then be the exceptional case that will not follow the open API interface.

Reply:
You could keep this , but if you are expecting to get a cookie injected then something like this will help:
![image](https://user-images.githubusercontent.com/17207278/115072208-92af3a00-9ef7-11eb-913d-3bcbe7230c73.png)

Reply:
ok Thanks. I will try this

Reply:
Thanks, this worked. updated in the pull request.